### PR TITLE
Dev re ex1

### PR DESCRIPTION
**Pull request by reckels**
You had misspelled my file as "stundent10" so I renamed it. That's what the file delete thing at the bottom is about.

### CODE REVIEW

**Comment by reckels** [comment]
just realized It put the wrong department name in. Gotta fix it!

File: src/main/java/com/xpanxion/java/assignments/student10/Worker10.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class Worker10 {
+
+    public void ex1() {
+        System.out.println("Ex. 1...");
+        var productList = DataAccess.getProducts();
+        var departmentList = DataAccess.getDepartments();
+
+        var newProductList = productList.stream().map(product -> {
+            var departmentId = product.getDepartmentId();
+            var departmentName = departmentList.get(departmentId).getName();
```
Comment (mburolla) [inline src/main/java/com/xpanxion/java/assignments/student10/Worker10.java]: This only works if the department Ids are in asc order, right?  What if they are not in order?

Comment (reckels) [reply src/main/java/com/xpanxion/java/assignments/student10/Worker10.java]: ahh that makes sense. k I'll figure out another way

Comment (reckels) [reply src/main/java/com/xpanxion/java/assignments/student10/Worker10.java]: ok I think I got it this time

**Comment by mburolla** [comment]
Yes, definitely fixed.  :-)

**Pull request merged**